backend/README.md

```diff
@@ -238,6 +238,8 @@ daemons to work correctly.
     python3 daemons/mail_notifier.py &
     # email fetcher (if email reply is enabled)
     python3 daemons/mail_fetcher.py &
+    # RQ worker for live messages
+    rq worker -q -w tracim_backend.lib.rq.worker.DatabaseWorker event
```

Comment:
Can you just explicit here what is -q and "event",  can't found the information in https://python-rq.org/docs/workers/ .

Reply:
I suspect there is somewhere in the config the same of the name of the queue used for TLM, I suggest to make this name configurable with a default value with something like tracim or tlm in name to avoid collisions with existing redis queu ?

Reply:
`-q` is to have quieter output from the worker, it is not mandatory.
`event` is the name of the queue the worker will listen for. It's possible to provide several depending on which queues tracim will use.